routes/items.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+const express = require('express');
+const passport = require('passport');
+const mongoose = require('mongoose');
+
+const { HttpError, NotFoundError, ValidationError } = require('../errors');
+
+const User = require('../models/user');
+const ShoppingList = require('../models/shopping-list');
+/*
+likely to add other models as routes are expanded upon
+- aislelocation model
+- category model
+*/
+
+const router = express.Router();
+const jwtAuth = passport.authenticate('jwt', { session: false });
+
+router.use(express.json());
+router.use(jwtAuth);
+
+//Add an item to a list
+router
+  .route('/:listId')
+  .get((req, res, next) => {
+    const { id: userId } = req.user;
+    const { listId } = req.params;
+    if (!mongoose.Types.ObjectId.isValid(listId)) {
+      throw new HttpError(422, `${listId} is not a valid ObjectId`);
+    }
+    ShoppingList.findById(listId)
+      .then(list => {
+        if (!list) {
+          throw new NotFoundError();
+        }
+        const items = list.items;
+        res.json({ items });
+      })
+      .catch(next);
+  })
+  .post((req, res, next) => {
```

Comment:
Probably want to put a blank line between get and post so it's clearer. Granted if we validate whether a user owns that list, we may need it then.